pkg/postgres/walker/walker.go

```diff
@@ -421,11 +421,12 @@ func handleStruct(ctx context, schema *Schema, original reflect.Type) {
 			schema.Children = append(schema.Children, childSchema)
 			handleStruct(
 				context{
-					searchDisabled: ctx.searchDisabled || searchOpts.Ignored,
-					ignorePK:       opts.IgnorePrimaryKey,
-					ignoreUnique:   opts.IgnoreUniqueConstraint,
-					ignoreFKs:      opts.IgnoreChildFKs,
-					ignoreIndex:    opts.IgnoreChildIndexes,
+					searchDisabled:     ctx.searchDisabled || searchOpts.Ignored,
+					ignorePK:           opts.IgnorePrimaryKey,
+					ignoreUnique:       opts.IgnoreUniqueConstraint,
+					ignoreFKs:          opts.IgnoreChildFKs,
+					ignoreIndex:        opts.IgnoreChildIndexes,
+					ignoreSearchLabels: opts.IgnoreSearchLabels,
```

Comment:
Was this option already implemented before and not work because this line was missing ?

Reply:
Did not work for array fields but worked for others.